src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -20,10 +20,12 @@
     using Domain.TournamentsAggregate;
     using GameResultConstants = Domain.Constants.GameResult;
 
-    /// <summary>
-    /// Defines an implementation of <see cref="IGameService"/> contract.
-    /// </summary>
+#pragma warning disable S1200 // Classes should not be coupled to too many other classes (Single Responsibility Principle)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)